packages/block-library/src/code/edit.native.js

```diff
@@ -20,7 +20,10 @@ import styles from './theme.scss';
 
 // Note: styling is applied directly to the (nested) PlainText component. Web-side components
 // apply it to the container 'div' but we don't have a proper proposal for cascading styling yet.
-export default function CodeEdit( { attributes, setAttributes, style } ) {
+export default function CodeEdit( props ) {
+	const { attributes, setAttributes } = props;
+	const { style } = attributes;
```

Comment:
It seems that `style` should be retrieved from `props`.

Reply:
my bad, addressed in 5f03b775a